lib/fileutils.rb

```diff
@@ -1294,20 +1301,22 @@ def remove_file(path, force = false)
   end
   module_function :remove_file
 
+  # Recursively removes the directory entry given by +path+,
+  # which should be the entry for a regular file, a symbolic link,
+  # or a directory.
   #
-  # Removes a directory +dir+ and its contents recursively.
-  # This method ignores StandardError if +force+ is true.
+  # Keyword arguments:
```

Comment:
This also needs to be fixed (not a keyword argument).

Reply:
Fixed.